nixos/doc/manual/release-notes/rl-2111.section.md

```diff
@@ -320,3 +320,5 @@ To be able to access the web UI this port needs to be opened in the firewall.
 - GNOME desktop environment now enables `QGnomePlatform` as the Qt platform theme, which should avoid crashes when opening file chooser dialogs in Qt apps by using XDG desktop portal. Additionally, it will make the apps fit better visually.
 
 - `rofi` has been updated from '1.6.1' to '1.7.0', one important thing is the removal of the old xresources based configuration setup. Read more [in rofi's changelog](https://github.com/davatorium/rofi/blob/cb12e6fc058f4a0f4f/Changelog#L1).
+
+- added derivation for free version of `dyalog apl` version '18.0.40684'. This is still proprietary software so the user must explicitly set `allowUnfree`.
```

Comment:
I am not sure if we should mention this in release notes to be honest. However, if someone decide we should do this, will be nice to run `./nixos/doc/manual/md-to-db.sh` to convert this to docbook as noted [here](https://nixos.org/manual/nixos/unstable/index.html#chap-contributing).